exercises/practice/pythagorean-triplet/src/pythagorean_triplet.cr

```diff
@@ -0,0 +1,5 @@
+module PythagoreanTriplet
+  def self.triplets_with_sum(sum : Int32) : Int32
```

Comment:
The output here doesn't seem right, shouldnt it return `Array(Array(Int32))`?

Reply:
Whoops, good catch!